src/components/Filter/OptionDisplay.tsx

```diff
@@ -97,29 +96,22 @@ export const OptionDisplay = ({
         tabIndex={-1}
       >
         {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
-        <div
-          className={classes.propagationStopper}
-          tabIndex={-1}
-          onClick={(event) => {
-            event.stopPropagation();
-          }}
-          onKeyDown={(event) => {
-            if (event.key !== 'Escape') {
-              event.stopPropagation();
-            }
+        <Checkbox
```

Comment:
Unit Tests are failing, I'll take await the review until they pass

Reply:
Yeah, this one isn't quite ready for review (hence why I haven't requested it yet, but I should probably have turned it back into a draft as well) 😅
I've already fixed the tests locally, but I'm awaiting response from the design system before committing since the checkbox component fails WCAG when setting external labels.

Reply:
ahh I thought you had forgotten to put on reviewers. It's possible to make draft PRs btw and also convert PR to draft PRs, to make it clear that a PR isn't ready for review

Reply:
Sorry didn't see you already mentioned about draft PR

Reply:
No worries 😊